### PR TITLE
RGO2018

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

**Review by MaxBaeten — APPROVED** [review]
I approve of all hardware changes
If  somebody approves of the travis, navigation and world_modelling changes we can merge it to master.

The track_unknown_space: true I agree with, since without we can't navigate trhough unknown space which is a big problem in the restaurant challenge

**Comment by MaxBaeten** [comment]
We can merge this if someone approves the world_modelling and navigation changes

**Comment by MaxBaeten** [comment]
What do we want to do with the Vieze node btw?

**Comment by MatthijsBurgh** [comment]
The vieze node is THE best code ever written. period.

File: launch/state_machines/challenge_final.launch [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
+			<arg name="faces_db" value="~/MEGA/data/$(optenv ROBOT_ENV rgo2018)/models/openface_ros/faces.pkl" />
+		</include>
+
+		<!-- Joke -->
+		<include file="$(find robot_launch_files)/launch/essentials/joke.launch">
+			<arg name="machine" value="amigo2" />
+		</include>
+
+		<!-- Vieze node -->
+		<node name="vieze_node" pkg="challenge_final" type="dirty_node.py" output="log" respawn="true" />
```
Comment (jlunenburg) [inline launch/state_machines/challenge_final.launch]: Rename...

File: parameters/navigation/global_costmap_gmapping.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 global_frame: /map
 robot_base_frame: /amigo/base_link
 rolling_window: false
-track_unknown_space: true
+track_unknown_space: false
```
Comment (jlunenburg) [inline parameters/navigation/global_costmap_gmapping.yaml:6]: This is quite a change.... @reinzor : do you approve?

Comment (MaxBaeten) [reply parameters/navigation/global_costmap_gmapping.yaml:6]: If he doesn’t we should make a separate one for the restaurant challenge

File: launch/state_machines/challenge_open.launch [hunk 3]
```diff
@@ -1,32 +1,36 @@
 <?xml version="1.0"?>
 
-<launch>
-
-    <arg name="robot_name" value="amigo" />
-    <arg name="robot_env" default="$(optenv ROBOT_ENV robotics_testlabs)" />
+<!-- Checkout README.md to see when to use this launch file -->
```
Comment (MatthijsBurgh) [inline launch/state_machines/challenge_open.launch]: This is copied from demo launch file, please remove.

File: launch/state_machines/challenge_open.launch [hunk 4]
```diff
@@ -1,32 +1,36 @@
...
-    </group>
+        <!-- Joke -->
+        <include file="$(find robot_launch_files)/launch/essentials/joke.launch" >
+            <arg name="machine" value="amigo2" />
+        </include>
 
-	<group if="$(optenv ROBOT_REAL false)">
-		<include file="$(find challenge_open)/launch/amigo_dependencies.launch">
-			<arg name="local_server_host" value="192.168.44.92"/>
+		<include file="$(find telegram_robot)/launch/conversation_engine_bot.launch" >
```
Comment (MatthijsBurgh) [inline launch/state_machines/challenge_open.launch:22]: Maybe not also move this launch file to robot_launch_files?

File: parameters/world_modeling/world_model.yaml [hunk 5]
```diff
@@ -58,17 +58,17 @@ plugins:
...
-#  frequency: 40
-#  parameters:
-#      laser_topic: /amigo/torso_laser/scan
-#      min_segment_size_pixels: 10
-#      world_association_distance: 0.4
-#      segment_depth_threshold: 0.2
-#      min_cluster_size: 0.2
-#      max_cluster_size: 1.0
-#      max_gap_size: 10
+- name: laser_integration
```
Comment (MatthijsBurgh) [inline parameters/world_modeling/world_model.yaml:61]: Do we want the laser integration in the default WM config?

Comment (Rayman) [reply parameters/world_modeling/world_model.yaml:61]: Yes, this enables us to do 'following'

File: launch/state_machines/challenge_open.launch (hunk 3 above)
Comment (MatthijsBurgh) [reply launch/state_machines/challenge_open.launch]: Done in d2604d92871e6ad69d179c017f523be79142ca42

File: parameters/navigation/global_costmap_gmapping.yaml (hunk 2 above)
Comment (MatthijsBurgh) [reply parameters/navigation/global_costmap_gmapping.yaml:6]: ping @reinzor

Comment (Rayman) [reply parameters/navigation/global_costmap_gmapping.yaml:6]: Rein: lijkt me goed

**Review by Rayman — APPROVED** [review]

**Pull request merged**